metadata.json

```
@@ -1,22 +1,27 @@
 {
-  "name": "srf-rkhunter",
-  "version": "1.0.1",
-  "author": "Gianni Carafa (SRF), Philipp Gassmann & Andreas Zuber (Puzzle)",
+  "name": "puppet-rkhunter",
+  "version": "2.0.0",
+  "author": "Vox Pupuli, Gianni Carafa (SRF), Philipp Gassmann & Andreas Zuber (Puzzle)",
   "license": "Apache-2.0",
   "summary": "Manage rkHunter installation and configuration",
-  "source": "git@github.com:mmz-srf/puppet-rkhunter.git",
-  "project_page": "https://github.com/mmz-srf/puppet-rkhunter",
-  "issues_url": "https://github.com/mmz-srf/puppet-rkhunter/issues",
+  "source": "git@github.com:voxpupuli/puppet-rkhunter.git",
```

Looks like in most other modules we give an https source.
```suggestion
  "source": "https://github.com/voxpupuli/puppet-rkhunter.git",
```